sdks/python/apache_beam/coders/coders_test.py

```diff
@@ -14,11 +14,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```

Comment:
<!--new_thread; commit:eee3edaa1aae1d97685c2465d8503c042ea46f15; resolved:0-->
Which of these is really necessary?  Do we use division and print in this file?